### PR TITLE
Basic translation for CA,ES

### PR DESCRIPTION
**Pull request by yopaseopor**
Basic translation for Catalan, Spanish

### CODE REVIEW

File: UI/i18n/Translations.ts [hunk 1]
```diff
@@ -910,74 +981,142 @@ export default class Translations {
...
-                    nl: "Wil je een eigen kaartthema, vraag dit <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>hier aan</a>"
+					ca: "Si vols que et fem una petició pròpia , demana-la <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>aquí</a>",
+					es: "Si quieres que te hagamos una petición propia , pídela <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>aquí</a>",
+                    nl: "Wil je een eigen kaartthema, vraag dit <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>hier aan</a>",
+                    fr: "Si vous voulez une autre carte thématique, demandez <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>ici</a>"
                 }),
 
                 streetcomplete: new T({
                     en: "Another, similar application is <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
+					ca: "Una altra aplicació similar és <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
```
Comment (pietervdvn) [inline UI/i18n/Translations.ts:1024]: I think you missed the spanish translation here

Comment (yopaseopor) [reply UI/i18n/Translations.ts:1024]: Fixed :) Thanks for your warning

File: UI/i18n/Translations.ts [hunk 2]
```diff
@@ -910,74 +981,142 @@ export default class Translations {
...
+					ca: "Crea la teva pròpia petició completa de MapComplete des de zero.",
+					ca: "Crea tu propia petición completa de MapComplete desde cero.",
+                    nl: "Maak je eigen MapComplete-kaart",
+                    fr: "Créez votre propre MapComplete carte"
                 })
             },
             readYourMessages: new T({
                 en: "Please, read all your OpenStreetMap-messages before adding a new point.",
+				ca: "Llegeix tots els teus missatges d\'OpenStreetMap abans d\'afegir nous punts.",
+				ca: "Lee todos tus mensajes de OpenStreetMap antes de añadir nuevos puntos.",
```
Comment (pietervdvn) [inline UI/i18n/Translations.ts:1040]: Here, you marked both of the texts to be catalan. WIth others I could figure out which one was which, but here I cant...

Comment (yopaseopor) [reply UI/i18n/Translations.ts]: fixed also

**Comment by pietervdvn** [comment]
YOU ARE AWESOME!

Thanks a lot for your effort! This is so cool!

The translations are merged and deployed, they will be online any minute now.

**Pull request merged**